src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -36,36 +40,36 @@
     private boolean isInstructorAllowedToDelete;
     private boolean isInstructorAllowedToEdit;
 
-    public FeedbackResponseCommentRow(FeedbackResponseCommentAttributes frc, String giverDisplay) {
+    public FeedbackResponseCommentRow(FeedbackResponseCommentAttributes frc, String giverDisplay,
+            Map<String, String> instructorEmailNameTable) {
+        this.instructorEmailNameTable = instructorEmailNameTable;
         this.commentId = frc.getId();
         this.giverDisplay = giverDisplay;
         this.createdAt = TimeHelper.formatDateTimeForComments(frc.createdAt);
-        this.editedAt = frc.getEditedAtText("Anonymous".equals(giverDisplay));
         this.commentText = frc.commentText.getValue();
+        this.commentGiverName = getCommentGiverNameFromEmail(giverDisplay);
+        this.questionId = frc.feedbackQuestionId;
```

Comment:
No, it's redundant. We need questionId in `Add Comment Form` but for that, we don't use this constructor. I'll remove it.